database/lwr_power_order.txt

```diff
@@ -0,0 +1,115 @@
+GRAND_GULF-1
+PEACH_BOTTOM-3
+PALO_VERDE-2
+PALO_VERDE-3
+PALO_VERDE-1
+PEACH_BOTTOM-2
+SOUTH_TEXAS-1
+SOUTH_TEXAS-2
+NINE_MILE_POINT-2
+SUSQUEHANNA-2
+SUSQUEHANNA-1
+SEABROOK-1
+PERRY-1
+CALLAWAY-1
+BROWNS_FERRY-3
+MILLSTONE-3
+COMANCHE_PEAK-1
+WOLF_CREEK
+BROWNS_FERRY-1
+BROWNS_FERRY-2
+COMANCHE_PEAK-2
+BRAIDWOOD-1
+HOPE_CREEK-1
+SALEM-1
+COOK-2
+WATERFORD-3
+WATTS_BAR-2
+BYRON-1
+CATAWBA-1
+BRAIDWOOD-2
+MCGUIRE-2
+MCGUIRE-1
+SALEM-2
+WATTS_BAR-1
+SEQUOYAH-1
+VOGTLE-2
+VOGTLE-1
+CATAWBA-2
+LASALLE-2
+SEQUOYAH-2
+DIABLO_CANYON-1
+LASALLE-1
+BYRON-2
+LIMERICK-1
+LIMERICK-2
+COLUMBIA
+DIABLO_CANYON-2
+FERMI-2
+TROJAN
+SAN_ONOFRE-3
+SAN_ONOFRE-2
+CLINTON-1
+ZION-2
+ZION-1
+INDIAN_POINT-3
+COOK-1
+INDIAN_POINT-2
+ANO-2
+ST._LUCIE-2
+ST._LUCIE-1
+SUMMER-1
+RIVER_BEND-1
+HARRIS-1
+NORTH_ANNA-1
+NORTH_ANNA-2
+BRUNSWICK-1
+BRUNSWICK-2
+QUAD_CITIES-2
+QUAD_CITIES-1
+BEAVER_VALLEY-1
+BEAVER_VALLEY-2
+DRESDEN-2
+DAVIS_BESSE-1
+HATCH-2
+FARLEY-2
+THREE_MILE_ISLAND-2
+DRESDEN-3
+CALVERT_CLIFFS-1
+HATCH-1
+FARLEY-1
+RANCHO_SECO-1
+MILLSTONE-2
+MAINE_YANKEE
+CRYSTAL_RIVER-3
+OCONEE-3
+CALVERT_CLIFFS-2
+OCONEE-2
+OCONEE-1
+SURRY-1
+SURRY-2
+TURKEY_POINT-3
+ANO-1
+TURKEY_POINT-4
+SHOREHAM
+THREE_MILE_ISLAND-1
+FITZPATRICK
+PALISADES
+COOPER
+ROBINSON-2
+PILGRIM-1
+MILLSTONE-1
+MONTICELLO
+OYSTER_CREEK
+NINE_MILE_POINT-1
+VERMONT_YANKEE
+DUANE_ARNOLD-1
+POINT_BEACH-1
+POINT_BEACH-2
+KEWAUNEE
+HADDAM_NECK
+GINNA
+PRAIRIE_ISLAND-1
+PRAIRIE_ISLAND-2
+FORT_CALHOUN-1
+SAN_ONOFRE-1
```

Comment:
I'm not quite familiar with your I/O formats, but consider turning this into a csv file so you can encapsulate the power associated with each reactor instead of needing to manually sort the reactors by their power.